samples/RaspberryPiInstructions.md

```diff
@@ -11,6 +11,7 @@ These steps have been tested on a RPi 2 and RPi 3 with Linux and Windows.
 
 * From the terminal/commandline create a folder named helloworld and go into it.
 * Run `dotnet new console -n helloworld`.
+* Change directory into `helloworld` and you can find helloworld.csproj file.
```

Comment:
You are right, I will update it. 
With recent SDK, if I ran `dotnet new console -n helloworld`, a new directory `helloworld` is created under current directory. I will update this paragraph.